packages/video_player_avplayer/tizen/src/plus_player.h

```diff
@@ -17,7 +17,8 @@
 
 class PlusPlayer : public VideoPlayer, public plusplayer::EventListener {
  public:
-  explicit PlusPlayer(flutter::BinaryMessenger *messenger, void *native_window);
+  explicit PlusPlayer(flutter::BinaryMessenger *messenger, void *native_window,
+                      std::string video_format);
```

Comment:
explicit PlusPlayer(flutter::BinaryMessenger *messenger, void *native_window,
                      std::string video_format);
 suggest change to:
 explicit PlusPlayer(flutter::BinaryMessenger *messenger, void *native_window,
                      std::string& video_format);

Reply:
fixed